### ISSUE TITLE
IndexController page title not translatable

### ISSUE THREAD

**Issue by Songworks** [DominicWatts/Special#11]
Hello there,

within `Xigen\Special\Controller\Index\Index::execute` you set the page title with `$resultPage->getConfig()->getTitle()->set('Special Offers');`.

Unfortunately it seems the method `Title::set()` sets the argument as pure string and it's not possible to translate it beforehand.
Could you please change the line to something like `$resultPage->getConfig()->getTitle()->set(__('Special Offers'));` (or whatever other method serves for translating within classes, like the `__()`)?

Or perhapts set the page title through the `view/frontend/layout/special_index_index.xml`:
```
<?xml version="1.0"?>
<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" layout="2columns-left" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
    <head>
        <title>Special Offers</title>
    </head>
    <body>
[...]
```
It should get automatically translated and the layout-xml is a lot easier to overwrite through a theme?

Cheers!

**Comment by Songworks** [comment]
Turns out setting the title in a special_index_index.xml within my theme already overwrites the title set within `Xigen\Special\Controller\Index\Index::execute`. So that is that I suppose. :)

**Comment by DominicWatts** [comment]
Fixed in latest release #12
